config/locales/en.yml

```diff
@@ -432,7 +432,8 @@ en:
     upload: Upload
   in_memoriam_html: In Memoriam.
   invites:
-    delete: Delete
+    delete: Force expire
```

Comment:
Maybe "deactivate"? "Force expire" is very technical

Reply:
👍 962e864